### ISSUE TITLE
Dark Mode Text Issues

### ISSUE THREAD

**Issue by Auroratide** [Auroratide/poke5e#592]
<img width="604" height="143" alt="cannot read" src="https://github.com/user-attachments/assets/50d3ac23-a2b8-4bc7-85f4-a3a5e88dcf58" />
